homeassistant/components/sensor/tibber.py

```diff
@@ -0,0 +1,99 @@
+"""
+Support for Tibber.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.tibber/
+"""
+import asyncio
+
+import logging
+
+from datetime import timedelta
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_ACCESS_TOKEN
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import dt as dt_util
+
+REQUIREMENTS = ['pyTibber==0.1.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_ACCESS_TOKEN): cv.string
+})
+
+ICON = 'mdi:currency-usd'
+SCAN_INTERVAL = timedelta(minutes=1)
+
+
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
+    """Set up the Tibber sensor."""
+    import Tibber
+    tibber = Tibber.Tibber(config[CONF_ACCESS_TOKEN],
+                           websession=async_get_clientsession(hass))
+    yield from tibber.update_info()
+    dev = []
+    for home in tibber.get_homes():
+        yield from home.update_info()
+        dev.append(TibberSensor(home))
+
+    async_add_devices(dev)
+
+
+class TibberSensor(Entity):
+    """Representation of an Tibber sensor."""
+
+    def __init__(self, tibber_home):
+        """Initialize the sensor."""
+        self._tibber_home = tibber_home
+        self._last_updated = None
+        self._state = None
+        self._device_state_attributes = None
+        self._unit_of_measurement = None
+        self._name = 'Electricity price  {}'.format(self._tibber_home.address1)
```

Comment:
Should there be two spaces after "price"?

Reply:
Thanks, fixed.
Tibber is available in Sweden too.